styles/prosilver/template/js/scoring_preview.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Wah. Seems I have commented the wrong js file. This is included on a different place. So this belongs to the other, I would guess. Seperate this when it is included on a different place is okay.